### DISCUSSION TITLE
mr-recv-coil-match-networks ready to publish

### DISCUSSION THREAD

**Discussion by thePlasm** [dtudk/Admin#6] [Ask for a sub-repo]
I have committed to mr-recv-coil-match-networks, which is now ready for a release. I have committed everything to the main branch (and squashed all of the commits). I am requesting for the repository to be made public.

Regards,
Rasmus

**Comment by thePlasm** [comment]
I would also like to add that I intend to publish this repo to Zenodo and as a PyPI package. I am not sure if Zenodo integration needs to be enabled per-repo, but if so, could that please be enabled for this repo?

**Comment by zerothi** [comment]
I have now changed it to public.

Let me search around for the zenodo thing what actually needs to be done. Currently Zenodo **is** enabled on the `dtudk` namespace, so I guess nothing should be done. Let me check.

**Comment by zerothi** [comment]
Ah, you will have to wait until Zenodo picks up the repo, then I can turn it on. So a bit of patience, perhaps first tomorrow

**Reply by zerothi** [reply]
Ok, zenodo picked it up, you should be able to make releases now!

I'll close this discussion, let me know if you have other inquiries.

**Reply by thePlasm** [reply]
Ok, thanks for your assistance.
